.storybook/stories/11-KaraokeMode.stories.tsx

```diff
@@ -61,24 +63,32 @@ export const KaraokeMode: React.FC<KaraokeModeProps> = args => {
 	useMediaListener('seeked', onSeek, listener);
 
 	useMediaListener(
-		'timeupdate',
+		'onTimeAlarm',
 		(e: TimeUpdateEvent) => {
-			if (currentPart.start - 0.2 < e.seconds && currentPart.end > e.seconds) {
-				return;
-			}
 			const res = findMatchingPartOrNext(transcriptRef.current, e.seconds);
 			setCurrentPart(res, 1);
 		},
 		listener,
 	);
+
+	// create alarms from transcript
+	const createAlarms = React.useCallback(() => {
+		for (let i = 0; i < mediaDuration; i++) {
```

Comment:
Can we use more meaningful variable names (e.g. `sec` instead of `i`)? If not possible, you can also keep `i`/`j` - it's just a bit harder to read.